netbox_librenms_plugin/static/netbox_librenms_plugin/js/librenms_sync.js

```
@@ -1400,6 +1404,39 @@ function initializeSyncFormSpinners() {
  */
+/**
+ * Wire the "Install Selected" form to collect checked module-table rows before submit.
+ * The form is separate from the table (to avoid nested forms), so we copy the
+ * selected checkbox values into hidden inputs just before the form is submitted.
+ * Guard against duplicate listeners on repeated HTMX swaps via a data attribute.
+ */
+function handleInstallSelectedSubmit() {
+    // Remove any previously-injected hidden inputs to avoid duplicates
+    const form = document.getElementById('install-selected-form');
+    if (!form) return;
+    form.querySelectorAll('input[data-injected-select]').forEach(el => el.remove());
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)